library/auth.inc

```diff
@@ -46,6 +46,7 @@ if (isset($_GET['auth']) && ($_GET['auth'] == "login") && isset($_POST['authUser
     else {
         $_SESSION['language_choice'] = 1;
     }
+    $_SESSION['language_direction'] = getLanguageDir($_SESSION['language_choice']  );
```

Comment:
It actually probably makes sense to set this here (to allow use of this in code that uses auth.inc) and to also support it in globals.php(which would then support use in the login screen for example).